src/Ztm.Zcoin.Rpc/ZcoinRpcClient.cs

```diff
@@ -426,7 +426,7 @@ async Task<ExodusTransaction> TryDecodeExodusTransaction(Transaction transaction
             {
                 infomation = await GetExodusTransactionAsync(transaction.GetHash(), CancellationToken.None);
             }
-            catch (Exception) // lgtm [cs/catch-of-all-exceptions]
+            catch (RPCException)
```

Comment:
This still dangerous due to it cannot distinguished what is the error. We should catch only if it is non-Exodus transaction error.